src/Schema/Builder.php

```diff
@@ -244,6 +249,26 @@ public function getIndexes($table)
             ];
         }
 
+        try {
+            $indexes = $collection->listSearchIndexes(['typeMap' => ['root' => 'array', 'array' => 'array', 'document' => 'array']]);
+            foreach ($indexes as $index) {
+                $indexList[] = [
+                    'name' => $index['name'],
+                    'columns' => match ($index['type']) {
+                        'search' => array_keys($index['latestDefinition']['mappings']['fields'] ?? []),
```

Comment:
Does this need better reporting for dynamic indexes, or is there no good way to express that given the expected format of `$indexList` elements?